types/bbob__parser/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for @bbob/parser 3.0
+// Project: https://github.com/JiLiZART/bbob
+// Definitions by: shme-e <https://github.com/shme-e>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export * from "./lexer";
+export * from "./parse";
+export * from "./Token";
+export * from "./utils";
```

Comment:
where are symbols from utils exported in original module?

Reply:
same for lever and Token

Reply:
if those are for types, importing them into index and re-exporting (with named exports) would be probably better (to avoid creating different export/imports compared to native module)

Reply:
thank you for the review! I added these exports to include the files as part of the project while developing however I don't think this is needed any more.

Reply:
hm, it seems the one in index is needed; not sure exactly how this should be treated, if a module is not exported at index or in the tree is there point to typing its functions at all?